reference/test/reorder/mc64_kernels.cpp

```diff
@@ -118,7 +119,7 @@ class Mc64 : public ::testing::Test {
                                           0., 2., 3., 0.}},
           final_dual_u{ref, I<real_type>{0., 1., -1., -2., 0., 0.}},
           final_distance{ref, I<real_type>{inf(), inf(), 1., 0., inf(), 1.}},
-          zero_tol{1e-14}
+          zero_tol{1e-4}
```

Comment:
perhaps `c * r<value_type>`?

Reply:
because it still passes the original tests with double or float, I keep 1e-4 for now